libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -121,13 +121,22 @@ public Bundle getTranslations() {
         configuration.locale = defaultLocale;
         Resources defaultResources = new Resources(getActivity().getAssets(), metrics, configuration);
 
-        for (Field stringField : R.string.class.getFields()) {
+        Class<?> R_String;
+        String maingPackage = getActivity().getApplication().getClass().getPackage().getName();
+
+        try {
+            R_String = getActivity().getApplication().getClassLoader().loadClass(maingPackage+".R$string");
+        } catch ( ClassNotFoundException ex ) {
+            return translations;
+        }
+
+        for (Field stringField : R_String.getDeclaredFields()) {
```

Comment:
Hmm interesting, I don't like hacky solutions generally :slightly_smiling_face: 
as it can just to stop working at some API levels. 
Did you test it with different Android API levels?

Reply:
I agree it's hacky, especially the part where it assumes that the `R` class is in the same package as the main Application class.
I'm not sure what you mean by different Android API levels though so I guess I didn't test that.
In theory we would need to go get the strings as an argument from the main app instead of trying to access a package we should not have to be aware of here.
It's starting to become a lot of changes for this "temporary" i18n solution though...

Reply:
Regarding Android API I am thinking to try this code with some devices that have **Lollipop**, **Marshmallow**... **Nougat**,,, and especially **EMUI** (Huawei) installed.